### ISSUE TITLE
Is plugin dead?

### ISSUE THREAD

**Issue by BaxAndrei** [rymate1234/JPanel#50]
Is this plugin development dead?

**Comment by ghost** [comment]
I believe so

**Comment by cupanoodle** [comment]
I'm unable to build on ubuntu and windows using gradlew build, have either of you had any success building the most recent source?

**Comment by ghost** [comment]
@cupanoodle Yes, it's not that hard. Refer to my comment on this other issue https://github.com/rymate1234/JPanel/issues/36#issuecomment-302974062

**Comment by Leymooo** [comment]
` Connect to rymatemc.co.uk:8081 [rymatemc.co.uk/176.9.7.154] failed: Connection refused: connect `

**Comment by ghost** [comment]
Sorry, I was not aware of that issue. It's been months since I last built the project.
Here is a direct link to the latest build of the project [PanelPlugin-1.0-SNAPSHOT.jar](https://skorrloregaming.com/PanelPlugin-1.0-SNAPSHOT.jar)
And here is what JPanel looks like after it has been properly installed and secured with ssl
![image](https://user-images.githubusercontent.com/12576827/41820665-0debcb7a-779b-11e8-9c5d-a4bf761244aa.png)

**Comment by ghost** [comment]
@Speentie8081 thank you dude SO MUCH!

**Comment by ghost** [comment]
@modgamer no problem

**Comment by Korinocho** [comment]
Hi, could you provide some new link? That is dead :3

**Comment by ghost** [comment]
Updated URI to download PanelPlugin-1.0-SNAPSHOT.jar
<https://ftp.rterminal.net/PanelPlugin-1.0-SNAPSHOT.jar>


Devon Casper
IT Systems Administrator

**Comment by mismith20** [comment]
Failed to open page.
Is there a new link?

**Comment by ghost** [comment]
Failed to open page.
Is there a new link?

**Comment by ghost** [comment]
Yes, sorry i'm doing a lot of changes to the server right now and I have to
keep the website down for the next 1-2 weeks.
http://www.mediafire.com/file/39zz1s9p63zw7or/PanelPlugin-1.0-SNAPSHOT.jar/file
